polyfactory/factories/sqlalchemy_factory.py

```diff
@@ -126,6 +129,11 @@ def get_type_from_column(cls, column: Column) -> type:
         column_type = type(column.type)
         if column_type in cls.get_sqlalchemy_types():
             annotation = column_type
+        elif issubclass(column_type, postgresql.ARRAY):
+            if type(column.type.item_type) in cls.get_sqlalchemy_types():  # type: ignore[attr-defined]
```

Comment:
```suggestion
        if column_type in (sqla_types := cls.get_sqlalchemy_types()):
            annotation = column_type
        elif issubclass(column_type, postgresql.ARRAY):
            if type(column.type.item_type) in sqla_types:  # type: ignore[attr-defined]
```
Saves calling that method twice.